lib/ansible/modules/network/ios/ios_vlan.py

```diff
@@ -212,34 +212,38 @@ def map_params_to_obj(module):
 
 
 def map_config_to_obj(module):
-    output = run_commands(module, ['show vlan'])
+    output = run_commands(module, ['show vlan brief'])
     lines = output[0].strip().splitlines()[2:-1]
 
     if not lines:
         return list()
-
     objs = list()
 
-    for l in lines:
-        splitted_line = l.strip().replace(",", "").split()
-        if splitted_line == []:
-            break
-        obj = {}
-        obj['vlan_id'] = splitted_line[0]
-        obj['name'] = splitted_line[1]
-        obj['state'] = splitted_line[2]
+    for index in range(len(lines)):
+        vlanID = lines[index][0:4].strip().replace(",", "")
```

Comment:
I'm not sure why you use exact index slices. Are they always this size?
I'd say that it's much safer to do `.split()` and that work with chunks:
```python
vlan_id, name, state, interfaces = [v.replace(',', '').strip() for v in current_line.split()]
```

please also note that in python we use so-called `snake_case` for variables, as opposed to `camelCase` (like in java or js)